10-inference.Rmd

```diff
@@ -601,7 +577,7 @@ boot1_dist
 summarise(boot1, mean = mean(price))
 ```
 
-Notice that our bootstrap distribution has a similar shape to the original
+Notice in Figure \@ref(fig:11-bootstrapping3) that our bootstrap distribution has a similar shape to the original
```

Comment:
again 15K here -- not a big deal but we should make them all have the same number of replicates for consistency